README.md

```diff
@@ -1,3 +1,77 @@
-# skywalking-swck
-
 Apache SkyWalking Cloud on Kubernetes
+============
+
+![](https://github.com/apache/skywalking-swck/workflows/Build/badge.svg?branch=master)
+
+<img src="http://skywalking.apache.org/assets/logo.svg" alt="Sky Walking logo" height="90px" align="right" />
+
+A bridge project between [Apache SkyWalking](https://github.com/apache/skywalking) and Kubernetes.
+
+SWCK is a platform for the SkyWalking user, provisions, upgrades, maintains SkyWalking relevant components, and makes them work natively on Kubernetes. 
+
+# Quick Start
+
+ 1. Go to the [download page](https://skywalking.apache.org/downloads/) to download latest release manifest. 
+ 1. Create the `skywalking-swck-system` namespace:
+ ```
```

Comment:
Click the review button, seems the list order is incorrect, maybe giving the code block some indents fixes it?

Reply:
> BTW, I think the namespace is kinda of long, and verbose, because swck contains “skywalking” already :)

It's from the repo's name, as a subproject, swck needs a prefix of the main project. and my preference is to keep it as it is.